test/ibmq/test_experiment.py

```diff
@@ -43,9 +43,17 @@ def setUpClass(cls):
         super().setUpClass()
         try:
             cls.provider = cls._setup_provider()    # pylint: disable=no-value-for-parameter
-            cls.experiments = cls.provider.experiment.experiments()
+            cls.experiments = cls.provider.experiment.experiments(tags=['qiskit-test'], limit=None)
             cls.device_components = cls.provider.experiment.device_components()
-        except Exception:
+            cls.BACKEND_NAME = 'ibmq_qasm_simulator'
```

Comment:
Does this need to be set on the class instance? Couldn't it just be a constant at module scope?

Reply:
[Updated! ](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1002/files#diff-6556bc5075d1aec9db8ea32abdc92d1064c71f0b43ac2a418486847932e9f0dcR65)Now a module constant.